internal/get.go

```diff
@@ -103,6 +103,9 @@ To get more details in output use one of the available flags.`,
 		if objectKindSupportsLabelsFlag(subCmd.Kind) {
 			registerLabelsFlag(sc, &get.labels)
 		}
+		if objectKindSupportsProjectSloFlag(subCmd.Kind) {
+			registerProjectSloFlag(sc, &get.slo, &get.project)
+		}
```

Comment:
I'm not sure that's needed :)
please use command `Extender` in the loop above instead